docs/index.html

```diff
@@ -9,7 +9,7 @@
 	<link rel='icon' type='image/png' href='favicon.png'>
 	<link rel='stylesheet' href='styles.css'>
 	<link rel='stylesheet' href='bundle.css'>
-	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/fontawesome.min.css">
```

Comment:
Can you please try the `all.min` file?
https://cdnjs.com/libraries/font-awesome

Reply:
Updated to all.min.css